### PR TITLE
Changed default TLD from .site to .suse

### PR DESCRIPTION
**Pull request by mchf**
bnc#922765

### CODE REVIEW

File: package/yast2-ca-management.changes [hunk 1]
```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Mon Jun 15 20:17:41 UTC 2015 - mfilka@suse.com
+
+- bnc#922765
+  - changed defautl TLD from .site to .suse
```
Comment (jreidinger) [inline package/yast2-ca-management.changes]: typo *default*

**Comment by jreidinger** [comment]
otherwise LGTM

**Comment by mvidner** [comment]
LGTM

**Comment by mvidner** [comment]
For reference, the change in the required yast2.rpm package is in yast/yast-yast2#370 (merged).

**Comment by mchf** [comment]
Thanks for reviews

**Comment by cwh42** [comment]
While porting to SLE12-SP1 if found another thing to change which you maybe have missed:
https://github.com/yast/yast-ca-management/blob/Code-11-SP4/src/ca_mgm_auto.ycp#L677
and some more places in that method.

**Comment by mchf** [comment]
great. Thank you

**Pull request merged**